src/scss/_bootstrap-variables.scss

```diff
@@ -27,6 +37,14 @@ $font-size-base: 1rem; // Assumes the browser default, typically `16px`
 $font-size-sm: $font-size-base;
 $font-size-lg: $font-size-base * 1.25;
 
+// Override the default headings font sizes.
+$h1-font-size: map-get($display-font-sizes, 1);
+$h2-font-size: map-get($display-font-sizes, 2);
+$h3-font-size: map-get($display-font-sizes, 3);
+$h4-font-size: map-get($display-font-sizes, 4);
+$h5-font-size: map-get($display-font-sizes, 5);
+$h6-font-size: map-get($display-font-sizes, 6);
```

Comment:
This is a really smart idea to simplify the code and tie the display font and header fonts together. Nice work!

Lines 49-54 are overwriting your great work though on these variables. Would you remove those later lines?